### PR TITLE
legit: add head version

### PR DESCRIPTION
**Pull request by wkentaro**
Closes kennethreitz/legit/issues/47

### CODE REVIEW

File: Library/Formula/legit.rb [hunk 1]
```diff
@@ -55,6 +56,10 @@ def install
 
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    if build.head?
+      man1.install "extra/man/legit.1"
+    end
```
Comment (bfontaine) [inline Library/Formula/legit.rb]: You can inline this: `man1.install "extra/man/legit.1" if build.head?`. Why is it done only when building `HEAD`?

Comment (wkentaro) [inline Library/Formula/legit.rb]: Because it has not yet released.

**Comment by wkentaro** [comment]
Updated.

**Comment by wkentaro** [comment]
revision 2?

**Comment by dunn** [comment]
We only add revisions for changes to the stable version, otherwise the majority of users (who run stable) would get an unnecessary upgrade.

**Comment by xu-cheng** [comment]
Why the formula has version 0.2.0 in URL while it's explicitly set as 0.1.0

**Comment by dunn** [comment]
@xu-cheng that was a mistake, fixed in b7f28f4aa9a696ff6ffac107b8c4daadab1b5a98

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**